actionpack/lib/abstract_controller/base.rb

```diff
@@ -96,7 +96,6 @@ def action_methods
             public_instance_methods(false))
 
           methods.map!(&:to_s)
-
```

Comment:
Can you restore this line break? Otherwise I think this is good to go.

Reply:
Sorry, I missed that